docs/examples/retrievers/vectara_auto_retriever.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Would it be better to print the resulting nodes + their metadata? Letting the nodes themselves go into stdout is a little cluttered </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9865/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>